luxonis_ml/data/parsers/solo_parser.py

```diff
@@ -158,136 +158,109 @@ def from_split(
-                                    }
-
-                            if (
-                                "SemanticSegmentationAnnotation"
-                                in annotation_types
-                            ):
-                                for anno in annotations:
-                                    if anno["@type"].endswith(
-                                        "SemanticSegmentationAnnotation"
-                                    ):
-                                        sseg_annotations = anno
-
-                                mask_fname = sseg_annotations["filename"]
-                                mask_path = os.path.join(
-                                    sequence_path, mask_fname
-                                )
-                                mask = cv2.imread(mask_path)
-
-                                for instance in sseg_annotations["instances"]:
-                                    class_name = instance["labelName"]
-                                    r, g, b, _ = instance["pixelValue"]
-                                    curr_mask = np.zeros_like(mask)
-                                    curr_mask[
-                                        np.all(mask == [b, g, r], axis=2)
-                                    ] = 1
-                                    curr_mask = np.max(
-                                        curr_mask, axis=2
-                                    )  # 3D->2D
-
-                                    yield {
-                                        "file": img_path,
-                                        "annotation": {
-                                            "type": "mask",
-                                            "class": class_name,
+
+                        if (
+                            "SemanticSegmentationAnnotation"
+                            in annotation_types
+                        ):
+                            for anno in annotations:
+                                if anno["@type"].endswith(
+                                    "SemanticSegmentationAnnotation"
+                                ):
+                                    sseg_annotations = anno
+
+                            mask_fname = sseg_annotations["filename"]
+                            mask_path = os.path.join(sequence_path, mask_fname)
+                            mask = cv2.imread(mask_path)
+
+                            for instance in sseg_annotations["instances"]:
+                                class_name = instance["labelName"]
+                                r, g, b, _ = instance["pixelValue"]
+                                curr_mask = np.zeros_like(mask)
+                                curr_mask[
+                                    np.all(mask == [b, g, r], axis=2)
+                                ] = 1
+                                curr_mask = np.max(curr_mask, axis=2)  # 3D->2D
+                                yield {
```

Comment:
Should include in the yield together with keypoints below?

Reply:
The same situation like COCO above.